### PR TITLE
Create group with Assigned PDL

### PR DESCRIPTION
**Pull request by AdwoodMicrosoft**
Create a group with a specified PDL from the SPO Admin Center.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/multi-geo-add-group-with-pdl.md [hunk 1]
```diff
@@ -14,11 +14,28 @@ description: "Learn how to create an Office 365 group with a specified preferred
 
 # Create an Office 365 Group with a specific PDL
 
-When users in a multi-geo environment create an Office 365 Group, the group preferred data location is automatically set to that of the user. If you need to create a group with a specific PDL, you can do that using the Exchange Online New-UnifiedGroup Microsoft PowerShell cmdlet. When you do this, both the group mailbox and SharePoint site associated with the group will be provisioned in the specified PDL.
+When users in a multi-geo environment create an Office 365 Group, the group preferred data location is automatically set to that of the user. Global, SharePoint, and Exchange Administrators can create groups in any region they select. 
 
-You must be a global administrator or a SharePoint Online or Exchange Online administrator to do this.
+If you need to create a group with a specific PDL, you can do that using from the Modern SharePoint Site Administration Experience or through the Exchange Online New-UnifiedGroup Microsoft PowerShell cmdlet. When you do this, both the group mailbox and SharePoint site associated with the group will be provisioned in the specified PDL.
```
Comment (JohanFreelancer9) [inline Enterprise/multi-geo-add-group-with-pdl.md]:
```suggestion
If you need to create a group with a specific PDL, you can do that using the Modern SharePoint Site Administration Experience or through the Exchange Online New-UnifiedGroup Microsoft PowerShell cmdlet. When you do this, both the group mailbox and SharePoint site associated with the group will be provisioned in the specified PDL.
```

File: Enterprise/multi-geo-add-group-with-pdl.md [hunk 2]
```diff
@@ -14,11 +14,28 @@ description: "Learn how to create an Office 365 group with a specified preferred
...
+Go to the Modern SharePoint Admin Center in the Geo location where you want to create the group site.
+
+For example:
+
+If you wan to create a group site in your Australtia location, you can go to https://ContosoAUS-admin.sharepoint.com/_layouts/15/online/AdminHome.aspx#/siteManagement
+
+1. Select "+ Create" 
+2. Follow the process to create a group site.
+
+Your group site will be provisioned in the geo location corresponding to the Modern SharePoint Admin Center from which you initiated the site creation request.
```
Comment (JohanFreelancer9) [inline Enterprise/multi-geo-add-group-with-pdl.md]:
```suggestion
Your group site will be provisioned in the Geo location corresponding to the Modern SharePoint Admin Center from which you initiated the site creation request. 
```

File: Enterprise/multi-geo-add-group-with-pdl.md [hunk 3]
```diff
@@ -14,11 +14,28 @@ description: "Learn how to create an Office 365 group with a specified preferred
...
 
-You must be a global administrator or a SharePoint Online or Exchange Online administrator to do this.
+If you need to create a group with a specific PDL, you can do that using from the Modern SharePoint Site Administration Experience or through the Exchange Online New-UnifiedGroup Microsoft PowerShell cmdlet. When you do this, both the group mailbox and SharePoint site associated with the group will be provisioned in the specified PDL.
 
-To create an Office 365 Group with the PDL that you specify, connect to Exchange Online PowerShell and pass the parameter *-MailBoxRegion* with the geo location code.
+To create an Office 365 Group with the PDL that you specify:
+
+From the Modern SharePoint Admin Center
+
+Go to the Modern SharePoint Admin Center in the Geo location where you want to create the group site.
```
Comment (JohanFreelancer9) [inline Enterprise/multi-geo-add-group-with-pdl.md]:
```suggestion
Go to the Modern SharePoint Admin Center in the geo location where you want to create the group site.
```

**Comment by mypil** [comment]
@AdwoodMicrosoft - please merge the commit on the copy/edit review done by JohanFreelancer9  or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@AdwoodMicrosoft - We would like to follow up. Please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@MikePlumleyMSFT - PR has been copy/edited and there are just some minor edits but we got no response from AdwoodMicrosoft to merge the commits so we leave it to your discretion whether or not to merge this PR. 

Thank you.

cc: @kenwith

**Pull request merged**